server/src/main/java/org/elasticsearch/common/Strings.java

```diff
@@ -917,38 +919,27 @@ public static boolean isNullOrBlank(@Nullable String s) {
         return s == null || s.isBlank();
     }
 
+    /**
```

Comment:
Should we make an effort to remove this method completely? Or is that too large of a change at this point?

Reply:
Gone